clang/test/CheckedCRewriter/checkedregionvariadicBUG.c

```diff
@@ -0,0 +1,58 @@
+// RUN: cconv-standalone -addcr  %s -- | FileCheck -match-full-lines --check-prefixes="CHECK" %s
```

Comment:
This test doesn't work? That's what I take from `BUG...` at the end.

Reply:
Yes, took out the variadic containing code until I fix the macro issue

Reply:
OK.